src/sentry/services/hybrid_cloud/auth/impl.py

```diff
@@ -64,11 +65,13 @@ def enable_partner_sso(
                 )
 
                 if user_id:
-                    sso_enabled.send_robust(
+                    organization_service.schedule_signal(
+                        sso_enabled,
                         organization_id=organization_id,
-                        user_id=user_id,
-                        provider=provider_key,
-                        sender=type(self) if sender is None else sender,
+                        args=dict(
+                            user_id=user_id,
+                            provider=provider_key,
```

Comment:
Do we want to retain `sender` at all? Not sure if it's relevant in the outbox based signal handler.

Reply:
I tried to retain it, but the RPC method complained of duplicate arguments. The sender isn't used in handler so we should be ok here.